web/src/components/TimeSlider.tsx

```diff
@@ -62,14 +60,19 @@ export const getTrackBackground = (
 export const getThumbIcon = (
   selectedIndex?: number,
   sets?: NightTimeSet[]
-): ThumbIconPath => {
+): ReactElement => {
+  const size = 14;
   if (selectedIndex === undefined || !sets || sets.length === 0) {
-    return 'slider-thumb.svg';
+    return <FaArrowsLeftRight size={size} pointerEvents="none" />;
```

Comment:
I might be biased here, but I think the existing one is much cleaner and nicer than the new FA one.

![Screenshot 2024-07-16 at 20 49 14](https://github.com/user-attachments/assets/ab8c43a7-4329-4588-8852-01c61b928951)

Reply:
We decided to switch to Lucide icons as you suggested so hopefully it looks good now 😬.